packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.test.js

```diff
@@ -56,4 +56,37 @@ describe('<SliderUnstyled />', () => {
     expect(element.getAttribute('styleProps')).to.equal(null);
     expect(element.getAttribute('theme')).to.equal(null);
   });
+
+  describe('prop: orientation', () => {
+    it('sets the orientation via ARIA', () => {
+      render(<SliderUnstyled orientation="vertical" />);
+
+      const slider = screen.getByRole('slider');
+      expect(slider).to.have.attribute('aria-orientation', 'vertical');
+    });
+
+    it('does nto set the orientation via appearance for WebKit browsers', function test() {
+      if (/jsdom/.test(window.navigator.userAgent) || !/WebKit/.test(window.navigator.userAgent)) {
+        this.skip();
+      }
+
+      render(<SliderUnstyled orientation="vertical" />);
+
+      const slider = screen.getByRole('slider');
+
+      expect(slider).to.have.property('tagName', 'INPUT');
+      expect(slider).to.have.property('type', 'range');
+      // Only relevant if we implement `[role="slider"]` with `input[type="range"]`
+      // We're not setting this by default because it changes horizontal keyboard navigation in WebKit: https://bugs.chromium.org/p/chromium/issues/detail?id=1162640
+      expect(slider).not.toHaveComputedStyle({ webkitAppearance: 'slider-vertical' });
```

Comment:
?
```suggestion
      expect(slider).not.toHaveComputedStyle({ WebkitAppearance: 'slider-vertical' });
```

Reply:
That's computed style i.e. a `CSSDeclaration` not what JSS is parsing.